### ISSUE TITLE
[Question] Build local development environment

### ISSUE THREAD

**Issue by JackyYangPassion** [apache/incubator-hugegraph-ai#11]
### Problem Type (问题类型)

others (please comment below)

### Before submit

- [X] I had searched in the [issues](https://github.com/apache/hugegraph-toolchain/issues) and found no similar issues.

### Environment (环境信息)

python 3.10
Pycharm
mac

### Your Question (问题描述)

# Use pycharm to run test.py and report an error
```
Traceback (most recent call last):
  File "/Users/yangjiaqi/Documents/hugegraph_concurrency/repo/incubator-hugegraph-ai/hugegraph-python/example/test.py", line 18, in <module>
    from src.client import PyHugeClient
ModuleNotFoundError: No module named 'src'

Process finished with exit code 1
```
# Root cause:
![image](https://github.com/apache/incubator-hugegraph-ai/assets/13795366/d0fff7bc-0993-48d5-be74-c49251103513)

# Solution:
![result](https://github.com/apache/incubator-hugegraph-ai/assets/13795366/d74049c9-339a-4ad7-b662-a888a7c636a0)

**Comment by simon824** [comment]
has been fix in #17 
<img width="371" alt="image" src="https://github.com/apache/incubator-hugegraph-ai/assets/18065113/2b79027c-370f-4966-8479-3a02523f3b5e">
or we can set env like: 
`export PYTHONPATH=$(pwd)/hugegraph-llm/src:$(pwd)/hugegraph-ml/src:$(pwd)/hugegraph-python-client/src`

**Comment by JackyYangPassion** [comment]
```
✅ 在 PyCharm 中配置 HugeGraph 项目的环境

打开项目
在 PyCharm 里打开 incubator-hugegraph-ai 这个目录。

选择解释器 (Interpreter)

打开 Preferences (⌘, 或 File → Settings)

找到 Project → Python Interpreter

点击右上角齿轮 ⚙️ → Add Interpreter

选择 Add Local Interpreter → Existing Environment

指定到你之前用 uv 创建的虚拟环境：

/完整路径/incubator-hugegraph-ai/.venv/bin/python


点击 OK，等待 PyCharm 加载依赖。
```
